apps/ensindexer/src/lib/plugin-helpers.ts

```diff
@@ -169,32 +125,32 @@ export interface PonderENSPlugin<PLUGIN_NAME extends PluginName, CONFIG> {
 }
 
 /**
- * An ENS Plugin's handlers are configured with ownedName and namespace.
+ * An PonderENSPlugin's handlers are provided runtime information about their respective plugin.
```

Comment:
```suggestion
 * A PonderENSPlugin's handlers are provided runtime information about their respective plugin.
```

Also, I'm not sure about the ending of this sentence? What does it mean for a plugin to be provided runtime information about their respective plugin? It sounds circular?

Reply:
the _handlers_ are provided runtime information — handlers are lazy-activated